tests/test_cli_compile.py

```diff
@@ -365,7 +323,9 @@ def test_locally_available_editable_package_is_not_archived_in_cache_dir(
 )
 @mark.parametrize(("generate_hashes",), [(True,), (False,)])
 @pytest.mark.network
```

Comment:
```suggestion
```

This could be removed now since the test able to run offline.

Reply:
Done.